dev_tools/configure_vscode_for_bazel.py

```diff
@@ -144,29 +180,45 @@ def get_new_config(executable_labels: set[str]) -> dict[str, Any]:
     }
 
 
-def save_new_config(new_config: dict[str, Any], config_location: Path, force: bool) -> None:  # noqa: FBT001
+def save_new_launch_config(new_config: dict[str, Any], config_location: Path, force: bool) -> bool:  # noqa: FBT001
     """Serializes the new_configuration to config_location.
 
     If the file already exists, asks for confirmation, unless force is set.
     """
-    if config_location.exists() and not force:
-        logging.warning("File %s already exists.", config_location.resolve())
-        confirm_or_abort("Do you want to overwrite it?")
-    config_location.write_text(json.dumps(new_config, indent=4))
-    logging.info("Saved new configuration to %s", config_location)
-
-
-def print_build_reminder(bazel_patterns: list[str]) -> None:
-    infix = " eg.:" if len(bazel_patterns) > 1 else ":"
-    logging.info(
-        "Remember to build the target(s) beforehand with%s\n\nbazel build --config=debug %s", infix, bazel_patterns[0]
-    )
-
-
-def update_launch_json(bazel_patterns: list[str], config_location: Path, force: bool) -> None:  # noqa: FBT001
-    executable_labels = find_executable_labels(bazel_patterns, force)
-    new_config = get_new_config(executable_labels)
-    save_new_config(new_config, config_location, force)
+    if confirm_config_overwrite(config_location, force):
+        config_location.write_text(json.dumps(new_config, indent=4))
+        logging.info("Saved new configuration to %s", config_location)
+        return True
+    return False
+
+
+def update_launch_json(bazel_patterns: list[str], config_location: Path, force: bool) -> bool:  # noqa: FBT001
+    if executable_labels := find_executable_labels(bazel_patterns, force):
+        new_config = get_new_launch_config(executable_labels)
+        return save_new_launch_config(new_config, config_location, force)
+    return False
+
+
+def update_cc_build_file(bazel_patterns: list[str], config_location: Path, force: bool) -> bool:  # noqa: FBT001
+    if confirm_config_overwrite(config_location, force):
+        sep = "," + "\n" + " " * 24
+        config_location.write_text(
+            textwrap.dedent(
+                f"""
+                load("@hedron_compile_commands//:refresh_compile_commands.bzl", "refresh_compile_commands")
```

Comment:
This assumes that the project is using hedron_compile_commands. We should highlight this in the readme.

Speaking about the readme, please document the new features there as well.